Dockerfile.dapper

```diff
@@ -62,19 +59,55 @@ RUN mkdir /usr/tmp && \
     mv /usr/tmp/helm /usr/bin/helm_v3 && \
     chmod +x /usr/bin/kustomize
 
-RUN curl -sLf ${!K3S_DOWNLOAD_URL} -o /usr/bin/k3s && \
-    chmod +x /usr/bin/k3s && \
-    mkdir -p /go/src/github.com/rancher/rancher/.kube && \
+# Set up K3s: copy the necessary binaries from the K3s image.
+COPY --from=rancher/k3s:v1.22.4-k3s1 \
```

Comment:
For maximum futureproofing, I would actually think we might want to directly consume the `k3s-root` archive that is where these userspace binaries all come from in the first place. 

Either that, or we could copy the entire `/bin` of the `rancher/k3s` image into a folder like `/usr/bin/k3s-bin` and then set the path in the container to `PATH=$PATH:/usr/bin/k3s-bin` and would have a lot less tech debt regarding binaries we have to maintain. 

There's a whole bunch of convuluted logic in place but more or less it all comes from https://github.com/k3s-io/k3s-root

https://github.com/k3s-io/k3s/blob/master/package/Dockerfile#L3-L5

I suppose we might only actually want a subset of the binaries -- I would be curious to see how you came to determine these are the specific binaries we need in the `rancher/rancher` image (I might just be missing it)

Reply:
That's an interesting idea, but I was trying to get them away from downloading artifacts in favor of `COPY --from` for everything.